metrics/prometheus.go

```diff
@@ -1746,6 +1751,54 @@ func NewPrometheusCollector(i infoProvider, f ContainerLabelsFunc, includedMetri
 		})
 	}
 
+	if includedMetrics.Has(container.PressureMetrics) {
```

Comment:
I can almost see these as also being nested under cpu/memory/disk metrics. I am not sure if there is precedent for this, but maybe add both a PressureMetrics for included metrics, as well as check the respective other metric? 

```

	if includedMetrics.Has(container.PressureMetrics) && includedMetrics.Has(container..CPUMetrics) {
report CPU pressure metrics
```

Reply:
@haircommander I couldn't find any precedent for this. I'll add a check for each of PSI resource (cpu, memory, io). I can't find a strong argument against or for adding such a check.

Reply:
I don't see why having pressure metrics needs to depend on other metrics. Each metric dataset is independent of each other.

Reply:
@SuperQ as PSI metrics are reported as part of the cpu/memory/io controller it might make sense to not report them when a user actively decides against getting metrics for one of these controllers.

It's a bit strange in the case of _io_ as it's reported for block io not only disk.

Like pointed out before I'm undecided about this, it might also make sense to defer this decision to a later point in time when a real use-case arises. This then raises the question which decision provides more backwards compatibility.

Reply:
we can always emit differently in the future, I think this makes sense for now.